packages/fast-components-react-msft/src/action-toggle/README.md

```diff
@@ -1,12 +1,12 @@
-# Action Toggle
+# Action toggle
 
-*Action toggle* should be used as a control on or around an interactive region of the page that affects the region or items within the region. It usually represents a switch between one of two states.
+Use *action toggle* to switch between two on screen binary states.
```

Comment:
I think `on screen` should be `on-screen`

Reply:
Or maybe just use "visible"

Reply:
I went with on-screen to avoid confusion with the visible state.